stories/scss/button/button-basic.stories.ts

```diff
@@ -45,12 +44,12 @@ function getTemplate(args: ButtonBasicStory): string {
 	const attributes = args.disabled ? `disabled="disabled"` : '';
 
 	return `
-	<button class="button ${classes}" ${attributes}>${args.label}</button>
+	<button class="button ${classes}" ${attributes}>Bouton</button>
```

Comment:
Why do you delete the dynamic text ? It allows to test the line breaks for example.

Reply:
I order to reduce the documentation size included in ZH as each attribute add a new entry in documentation table. 
What if we keep it only inside QA stories? 🤔